Lombiq.Tests.UI/Services/UITestContext.cs

```diff
@@ -214,7 +214,7 @@ public async Task TriggerAfterPageChangeEventAndRefreshAtataContextAsync()
     /// <summary>
     /// Changes the current tenant context to the Default one. Note that this doesn't navigate the browser.
     /// </summary>
-    public void ChangeCurrentTenantToDefault() => SwitchCurrentTenant("Default", string.Empty);
+    public void SwitchCurrentTenantToDefault() => SwitchCurrentTenant("Default", string.Empty);
```

Comment:
This is a breaking change that most possibly doesn't affect too many. But if we merge this before https://github.com/Lombiq/Open-Source-Orchard-Core-Extensions/pull/454 then no issue.